minorminer/utils/embeddings.py

```diff
@@ -131,18 +136,32 @@ def visualize_embeddings(
     )
 
 
-def shuffle_graph(T, prng=None):
-    """Shuffle the node and edge ordering of a networkx graph."""
+def shuffle_graph(G: nx.Graph, prng=None) -> nx.Graph:
+    """Shuffle the node and edge ordering of a networkx graph.
+
+    For embedding methods that operate as a function of the node or edge
+    ordering this can force diversification in the returned embeddings. Note
+    that special orderings that encode graph structure or geometry may enhance
+    embedder performance (shuffling may lead to longer search times).
+
+    Args:
+        G (nx.Graph): A networkx graph
+        prng (np.random.Generator, optional): When provided, is used to shuffle
+            the order of nodes and edges in the source and target graph. This
+            can allow sampling from otherwise deterministic routines.
+    Returns:
+        nx.Graph: The same graph with modified node and edge ordering.
+    """
     if prng is None:
         prng = np.random.default_rng()
```

Comment:
numpy handles `None` automatically
```suggestion
    prng = np.random.default_rng(prng)
```

Reply:
We should change the name of the input argument to seed. Allow types: Union[int, np.random.RandomState, np.random.Generator], and document that the seed can be an integer used to seed a numpy random generator,  or a numpy random number generator."

Reply:
I used to put more detailed type hinting but somethings it makes the code too long.